requirements/requirements.txt

```diff
@@ -1,4 +1,5 @@
 pandas>=1.1.5
+pandas<2.2.0
```

Comment:
Alex suggested that our process should be:
1. Pandas version comes out
2. We test it
3. We move this threshold

Rather than the current process of:
1. Pandas version comes out
2. Something breaks
3. Someone complains (hopefully)
4. We fix